lib/improver/tests/set_up_test_cubes.py

```diff
@@ -366,9 +366,19 @@ def set_up_probability_cube(data, thresholds, variable_name='air_temperature',
         attributes['relative_to_threshold'] = relative_to_threshold
 
     if variable_name.startswith('probability_of_'):
-        name = variable_name
+        name_prefix = ''
     else:
-        name = 'probability_of_{}'.format(variable_name)
+        name_prefix = 'probability_of_'
```

Comment:
You could also update the Docstring where it mentions the probability_of_X convention

Reply:
Updated.